packages/docs/operators/_change_case.yaml

```diff
@@ -0,0 +1,287 @@
+# Copyright 2020-2021 Lowdefy, Inc
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+_ref:
+  path: templates/operators.yaml.njk
+  transformer: templates/operatorsMethodTransformer.js
+  vars:
+    pageId: _change_case
+    pageTitle: _change_case
+    filePath: operators/_change_case.yaml
+    description: |
+      The `_change_case` operator uses the [`change-case`](https://www.npmjs.com/package/change-case) package to transform an input between camelCase, PascalCase, Capital Case, snake_case, param-case, CONSTANT_CASE and others.
+    arguments: |
+      The `_change_case` operator takes `on` and `options` as arguments.
+      The `on` argument is the input to be transformed and can be a `string`,  `array` or `object`.
+      The `options` argument is an object with the following properties:
+        - `splitRegexp`: The regular expression used to split `on` into word segments.
```

Comment:
Can we add tests for these regex options?

When passing in reg expressions to operators you need to test for the option and create a Regex from the input string to pass to the lib. 

Have a look at the implementation for `_regex`.